Update credential documentation (#1428)
Cray-HPE/docs-csm: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md

**NOTE:** This procedure does not provision Slingshot switch BMCs (RouterBMCs). Slingshot switch BMC default credentials must be changed using the procedures in the Slingshot product documentation. To update Slingshot switch BMCs, refer to "Change Rosetta Login and Redfish API Credentials" in the *Slingshot Operations Guide (> 1.6.0)*.

This procedure provisions only the default Redfish `root` account passwords. It does not modify Redfish accounts that have been added after an initial system installation.

### Prerequisites

> - The hms-discovery Kubernetes cron job has been disabled.

Reviewer: ```suggestion
- The `hms-discovery` Kubernetes cron job has been disabled.
```

    > ncn-m001# cray hsm inventory redfishEndpoints update BMC_XNAME --user root --password ${CRED_PASSWORD}
    > ```

> 3. Restart the hms-discovery Kubernetes cron job.

Reviewer: ```suggestion
3. Restart the `hms-discovery` Kubernetes cron job.
```

   ```bash
   ncn-m001# kubectl -n services patch cronjobs hms-discovery -p '{"spec" : {"suspend" : false }}'
   ```

>    After 2-3 minutes the hms-discovery cron job will start to power on all of the currently powered off compute slots.

Reviewer: ```suggestion
   After 2-3 minutes, the `hms-discovery` cron job will start to power on all of the currently powered off compute slots.
```

> 4. Wait for compute slots to be pwoered on and for HSM to re-discover the updated RedfishEndpoints:

Reviewer: ```suggestion
4. Wait for compute slots to be pwoered on and for HSM to re-discover the updated Redfish endpoints.
```

    ```bash
    ncn-m001# sleep 300
    ```

> 5. Wait for all updated Redfish endpoints to become `DiscoverOK`:

Reviewer: ```suggestion
5. Wait for all updated Redfish endpoints to become `DiscoverOK`.
```

>     The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their last Discovery Status.

Reviewer: ```suggestion
    The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `LastDiscoveryStatus`.
```

    > ncn-m001# cray hsm inventory redfishEndpoints update BMC_XNAME --user root --password ${CRED_PASSWORD}
    > ```

3. Restart the `hms-discovery` Kubernetes cron job.
   ```bash
   ncn-m001# kubectl -n services patch cronjobs hms-discovery -p '{"spec" : {"suspend" : false }}'
   ```
   After 2-3 minutes, the `hms-discovery` cron job will start to power on all of the currently powered off compute slots.

4. Wait for compute slots to be pwoered on and for HSM to re-discover the updated Redfish endpoints.
    ```bash
    ncn-m001# sleep 300
    ```

5. Wait for all updated Redfish endpoints to become `DiscoverOK`.

>     The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `lastDiscoveryStatus`.

Reviewer: ```suggestion
    The following Bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `lastDiscoveryStatus`.
```

Author: Grr, I changed that!
